examples/typescript-backbone/js/app.ts

```diff
@@ -261,6 +285,28 @@ class TodoView extends Backbone.View {
 
 }
 
+// Todo Router
+// -----------
+
+class TodoRouter extends Backbone.Router {
+
+	routes = {
+		'*filter': 'setFilter'
+	};
+
+	constructor () {
+		super();
+		(<any>this)._bindRoutes();
+	}
+
+	setFilter (param: string) {
```

Comment:
Yep, glad to do that. I think the master jscs style recommends the opposite but it can't be enforced on TypeScript. Honestly I prefer no space anyway. :bike: :house: